commands/settings.js

```diff
@@ -0,0 +1,107 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { reply, defer } = require('../helpers/interactionHelper')
+const { getGuild, saveGuild } = require('../helpers/guildData')
+const { verifyAdminAsync } = require('./admin')
+const { MessageEmbed } = require('discord.js')
+const { foemp } = require('../helpers/foemp')
+
+const setMemberNotificationChannel = async interaction => {
```

Comment:
the way we use them in this project, an arrow function behaves no different than a regular function.
[here you can see what differences there are between the two.](https://medium.com/geekculture/regular-vs-arrow-function-1f8140fbcece)

the motivation why I use arrow functions is because I'm used to them. It's my programming style preference 😅

Reply:
I'm not really against it, and as far as I'm concerned it is not necessery to block the PR on this point.
However I do think it would be better to see which syntax has the preference of the team as I dislike mixing them :D

Reply:
i reverted them to normal functions :)